inlang/source-code/paraglide/paraglide-js/docs/usage.md

```diff
@@ -130,6 +129,39 @@ m.hello() // Hello world!
 
 The [language tag](https://www.inlang.com/m/8y8sxj09/library-inlang-languageTag) is a global function. This means that on the server it is _shared_ accross requests. In order to avoid the langauge from one request being overwritten by another request you need to use _getter function_ that returns the language for the _current request_. A good way to implement this is using [`AsyncLocalStorage`](https://nodejs.org/api/async_context.html).
 
+**Bad** Example:
```

Comment:
I would work with some emojies (⚡️, ❌, ⛔️) to make super clear that this is wrong. And maybe make the whole title bold. In the future, we can work on a code comparison component.